packages/emails/src/components/LinkIcon.tsx

```diff
@@ -1,11 +1,17 @@
-import { BASE_URL, IS_PRODUCTION } from "@calcom/lib/constants";
-
-export const LinkIcon = ({ secondary }: { secondary?: boolean }) => (
+export const LinkIcon = ({ secondary, iconName }: { secondary?: boolean; iconName: string }) => (
   <img
-    src={IS_PRODUCTION ? BASE_URL + "/emails/linkIcon.png" : "https://app.cal.com/emails/linkIcon.png"}
-    srcSet={IS_PRODUCTION ? BASE_URL + "/emails/linkIcon.svg" : "https://app.cal.com/emails/linkIcon.svg"}
+    src={
+      IS_PRODUCTION ? WEBAPP_URL + `/emails/${iconName}.png` : `https://app.cal.com/emails/${iconName}.png`
```

Comment:
I am not sure why we are keeping this IS_PRODUCTION check.
Shouldn't we just use `${WEBAPP_URL}/emails/${iconName}.png`